tests/openwisp2/sample_config/migrations/0004_redundant_migration.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.22 on 2021-05-07 13:43
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sample_config', '0003_name_unique_per_organization'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='vpn',
+            name='subnet',
+            field=models.ForeignKey(
+                blank=True,
+                help_text='Subnet IP addresses used by VPN clients, if applicable',
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to=settings.OPENWISP_IPAM_SUBNET_MODEL,
+                verbose_name='Subnet',
+            ),
```

Comment:
I don't understand why this file being created. All the parameters are same as in 
https://github.com/openwisp/openwisp-controller/blob/8ced25b289aaac4bec5d78fda8f3fb816c8237b0/tests/openwisp2/sample_config/migrations/0001_initial.py#L325-L336

Reply:
@pandafy if I am not wrong, this happens due to this
https://github.com/openwisp/openwisp-controller/blob/8ced25b289aaac4bec5d78fda8f3fb816c8237b0/tests/openwisp2/sample_config/migrations/0003_name_unique_per_organization.py#L29-L40

Here, `on_delete`  is different from these two migrations :thinking:

Reply:
Thank you for catching this Kapil! 🙌🏼